auto3dseg/algorithm_templates/segresnet/scripts/segmenter.py

```diff
@@ -840,6 +842,52 @@ def parse_input_config(
 
         return config
 
+
+    def config_save_updated(self, save_path = None):
+
+        if self.config["auto_scale_allowed"]:
+            #reload input config
+            config = ConfigParser.load_config_files(self.config_file)
+            parser = ConfigParser(config=config)
+            parser.update(pairs=self.override)
+            config = parser.config
+
+            config["batch_size"] = self.config["batch_size"]
+            config["roi_size"] = self.config["roi_size"]
+            config["num_crops_per_image"] = self.config["num_crops_per_image"]
+
+            if "init_filters" in self.config["network"]:
+                config["network"]["init_filters"] = self.config["network"]["init_filters"]
+
+            if save_path is None:
+                save_path = self.config_file
+
+            if self.global_rank==0:
+                print(f"Re-saving main config to {save_path}.")
+                ConfigParser.export_config_file(config, save_path, fmt="yaml", default_flow_style=None, sort_keys=False)
```

Comment:
Maybe it is good to keep the original as a copy for record?

Reply:
We've discussed it and agreed to do it this way now for all algos.

Reply:
Okay, and thanks!